llm4pddl/utils.py

```diff
@@ -220,6 +220,15 @@ def minify_pddl_problem(problem: str) -> str:
         partially_flattened = new_problem.replace(')\n(', ')(')
         # Removing new lines in init, a space is needed between
         new_problem = partially_flattened.replace('\n', ' ')
+        # Add new lines for the question and answer tokens
+        new_problem = new_problem.replace('Q: ', 'Q:')
+        new_problem = new_problem.replace('A: ', 'A:')
```

Comment:
should I use `utils.LLM_ANSWER_TOKEN` instead of `A` here? and same with Q?

Reply:
yes please!